test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,19 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
-
-  test("returns Buzz when passed a multiple of 5", () => {});
-
-  test("returns FizzBuzz when passed a multiple 3 and 5", () => {});
-
-  test("returns the number when it isn't a multiple of 3 or 5", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz');
+  });
+  test("returns Buzz when passed a multiple of 5", () => {
+    expect(fizzBuzz(5)).toEqual('Buzz');
+  });
+  test("returns FizzBuzz when passed a multiple 3 and 5", () => {
+    expect(fizzBuzz(15)).toEqual('FizzBuzz');
+    expect(fizzBuzz(30)).toEqual('FizzBuzz');
+    expect(fizzBuzz(90)).toEqual('FizzBuzz');
+  });
+  test("returns the number when it isn't a multiple of 3 or 5", () => {
+    expect(fizzBuzz(8)).toEqual(8);
```

Comment:
This is great as you've done multiple inputs so you know the answer isn't hardcoded to return 8.

Reply:
It also helps you spot if your code works for only some cases, but not all.